sdk/cognitivelanguage/ai-language-text/test/public/expectations.ts

```diff
@@ -1695,15 +1672,13 @@ export const expectation20: AnalyzeBatchResult[] = [
               { name: "MDR", entityId: "10021731" },
               { name: "MSH", entityId: "D007223" },
               { name: "NCI", entityId: "C27956" },
-              { name: "NCI_FDA", entityId: "C27956" },
-              { name: "NCI_NICHD", entityId: "C27956" },
               { name: "SNOMEDCT_US", entityId: "133931009" },
             ],
             text: "Baby",
-            category: "Age",
+            category: "FamilyRelation",
```

Comment:
hmm, do you think a family relation implies there is another related entity, parent, that should be linked here?

Reply:
Interesting finding! may be it is a good idea to ask @gils-ms about it?